src/features/CityDetails/CityDetails.tsx

```diff
@@ -0,0 +1,55 @@
+import { StyleSheet, View } from 'react-native';
+import Row from 'components/Row/Row';
+import TemperaturePill from 'components/TemperaturePill/TemperaturePill';
+import WeatherIcon from 'components/WeatherIcon/WeatherIcon';
+import { ScrollView, Text } from 'react-native';
+import MainWeatherInfo from './MainWeatherInfo';
+import Loader from 'components/Loader/Loader';
+import { TItemSchema } from 'api/index';
+
+interface Props {
+  loading: boolean;
+  data?: TItemSchema;
```

Comment:
https://tkdodo.eu/blog/optional-vs-undefined#optional-properties

```suggestion
  data: TItemSchema | undefined;
```